src/lib.rs

```diff
@@ -502,6 +502,36 @@ pub mod damage {
     //!
     //! Accessible with the `damage` cargo feature.
     include!(concat!(env!("OUT_DIR"), "/damage.rs"));
+
+    impl NotifyEvent {
+        /// Returns the raw level/continuation byte (2nd byte of the on-wire structure).
+        fn raw_level(&self) -> u8 {
+            // level / continuation byte is the 2nd byte of the on-wire structure
+            unsafe { *(self.wire_ptr().add(1) as *const u8) }
```

Comment:
[`std::ptr::read_unaligned`](https://doc.rust-lang.org/std/ptr/fn.read_unaligned.html) must be used here